lib/tasks/grade.rake

```diff
@@ -1,4 +1,120 @@
-desc "Grade project"
-task grade: :environment do
-  puts `bundle exec rspec --order default --format documentation --format j --out test_output.json --format html --out test_output.html`
+# desc "Grade project"
+# task grade: :environment do
+#   puts `bundle exec rspec --order default --format documentation --format j --out test_output.json --format html --out test_output.html`
+# end
+
+require 'json'
+require 'yaml'
```

Comment:
Is this line necessary? I thought 'yaml' would be loaded by Rails automatically.